### ISSUE TITLE
False positive: empty_line_after_outer_attr 3+ empty lines enum

### ISSUE THREAD

**Issue by alexheretic** [rust-lang-nursery/rust-clippy#2475]
Using 5e6342d4ae1e4e8e90c8498a6174b95e584142a6 _empty_line_after_outer_attr_ warnings appear when a simple enum has 3+ empty lines. Doesn't occur with 1 or 2 empty lines.
```rust
#[derive(Clone, Copy)]
pub enum Foo {
    Bar1,

    Bar2,

    Bar3,

    Bar4,
}
```
```
warning: Found an empty line after an outer attribute. Perhaps you forgot to add a '!' to make it an inner attribute?
 --> src/main.rs:2:1
  |
2 | pub enum Foo {
  | ^
  |
  = note: #[warn(empty_line_after_outer_attr)] on by default
  = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.186/index.html#empty_line_after_outer_attr
```

**Comment by oli-obk** [comment]
cc @phansch

**Comment by phansch** [comment]
Thanks, I will take a look later today!
